pkg/agent/manager/cache/cache_test.go

```diff
@@ -644,6 +620,183 @@ func TestSubscriberNotNotifiedOnOverlappingSVIDChanges(t *testing.T) {
 	assertNoWorkloadUpdate(t, sub)
 }
 
+func TestSVIDCacheExpiry(t *testing.T) {
+	clk := clock.NewMock()
+	cache := newTestCacheWithConfig(10, 1*time.Minute, clk)
+
+	clk.Add(1 * time.Second)
+	foo := makeRegistrationEntry("FOO", "A")
```

Comment:
for readability, I think it would be helpful to keep foo/bar naming consistent; so changing `subA` and selectors for instance to `subFoo` and so on

Reply:
I'll do it in follow-up PR. This one is getting too big for variable renaming.